src/app/clusters/keypad-input-server/keypad-input-server.cpp

```diff
@@ -39,11 +39,37 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
+#include <app/common/gen/enums.h>
 #include <app/util/af.h>
 
-bool emberAfKeypadInputClusterSendKeyCallback(chip::app::Command * commandObj, unsigned char)
+EmberAfKeypadInputStatus keypadInputClusterSendKey(EmberAfKeypadInputCecKeyCode keyCode);
+
+static void sendResponse(chip::app::Command * command, EmberAfKeypadInputStatus keypadInputStatus)
+{
+    static_assert(std::is_same<std::underlying_type_t<EmberAfKeypadInputStatus>, uint8_t>::value, "Wrong enum size");
```

Comment:
Do we still need this assert?  The TLV doesn't care what size this is.

Reply:
Removed.